OrbitBase/include/OrbitBase/Platform/Linux/Profiling.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_BASE_PLATFORM_LINUX_PROFILING_H_
+#define ORBIT_BASE_PLATFORM_LINUX_PROFILING_H_
```

Comment:
looking at the functions - this should be called something like Utils or split into MonotonicTime.h (Native?)ThreadUtils.h maybe?

Reply:
I would avoid a very generic "Utils" but I agree on renaming to something more descriptive and possibly splitting.